plugins/package-managers/bundler/src/main/kotlin/Model.kt

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.bundler
+
+import com.charleskorn.kaml.Yaml
+import com.charleskorn.kaml.YamlConfiguration
+import com.charleskorn.kaml.YamlNamingStrategy
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+internal val YAML = Yaml(
+    configuration = YamlConfiguration(strictMode = false, yamlNamingStrategy = YamlNamingStrategy.SnakeCase)
+)
+
+/**
+ * The information for a Gem as typically defined in a ".gemspec" file, see
+ * - https://guides.rubygems.org/specification-reference/
+ * - https://github.com/rubygems/rubygems/blob/95128500fba28e3bb03d3bbfcf8536d00a893035/lib/rubygems/specification.rb#L19-L40
+ */
+@Serializable
+internal data class GemSpec(
+    /** The Gem’s name. */
+    val name: String,
+
+    /** The Gem’s specified version. */
+    val version: SpecVersion,
+
+    /** A long description of this Gem. */
+    val description: String? = null,
+
+    /** A short summary of this Gem’s description. */
+    val summary: String? = null,
+
+    /** The URI of this Gem’s homepage. */
+    val homepage: String? = null,
+
+    /** A list of authors for this Gem. */
+    val authors: List<String> = emptyList(),
+
+    /** The license(s) for the Gem. Each license must be a short name, no more than 64 characters. */
+    val licenses: List<String> = emptyList(),
+
+    /** The list of specified dependencies. */
+    val dependencies: List<SpecDependency> = emptyList()
+)
+
+@Serializable
+internal data class SpecVersion(
+    /** The version string, containing numbers and periods, such as "1.0.0.pre" for a prerelease. */
+    val version: String
+)
+
+@Serializable
+internal data class SpecDependency(
```

Comment:
Consider making `SpecDependency` and `SpecVersion` nested into `GemSpec`. 
This allows ommiting the prefix and spearates from the below API model classes.